src/Equinor.ProCoSys.IPO.MainApi/MainApi/Person/ProCoSysPerson.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Equinor.ProCoSys.IPO.ForeignApi.MainApi.Person
+{
+    public class ProCoSysPerson
+    {
+        public string AzureOid { get; set; }
+
+        public string UserName { get; set; }
```

Comment:
If we are adding userName here, it should be added to the persons under FunctionalRole as well

Reply:
Good catch! I'll add it to persons under functionalrole as well 👍